dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
I understand the problem now (the ability to ensure a given element always has a focus ring applied), however what I don't understand _why_ this polyfill needs to account for that when this can be accomplished by authors leveraging the natural cascade and/or specificity rules in CSS.

```
// hide focus by default
:focus {
  outline: none;
}
// focus ring for keyboard navigation
.focus-ring {
  outline: solid 2px blue;
}

// always show focus outline around a specifc control
button.default-action:focus {
  outline: solid 2px blue;
}
```

This way the polyfill is designed to be with-the-grain of the eventual pseudo class implementation.